src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/CSharpTagHelperCodeRenderer.cs

```diff
@@ -202,6 +202,13 @@ protected virtual string GenerateUniqueId()
                     // First attribute wins, even if there's duplicates.
                     var attributeValueChunk = matchingAttributes.First().Value;
 
+                    // Minimized attributes are not valid for bound attributes. There will be an error logged for this
+                    // bound attribute already so we can skip.
```

Comment:
Please make this comment a bit more informative, describing what logs the error.  "There will be" is just too vague.